Rosetta API implementation (#7695)
cosmos/cosmos-sdk: contrib/rosetta/README.md

# rosetta

> Contains the files required to run the rosetta CI. It builds a simapp based on the current codebase.

Reviewer: ```suggestion
This directory contains the files required to run the rosetta CI. It builds `simapp` based on the current codebase.
```
